packages/twenty-server/.env.example

```diff
@@ -42,6 +42,7 @@ FRONTEND_URL=http://localhost:3001
 # EXCEPTION_HANDLER_DRIVER=sentry
 # SENTRY_ENVIRONMENT=main
 # SENTRY_RELEASE=latest
+# APP_VERSION=1.2.3
```

Comment:
do we want to keep it here? I would only put it .env.example of docker-compose

Reply:
For anyone working on the upgrade command this would be a required env var, I'm not sure